support/cas-server-support-webauthn/src/main/java/org/apereo/cas/config/WebAuthnConfiguration.java

```diff
@@ -277,7 +270,9 @@ public WebAuthnServer webAuthnServer(
                 .appId(appId)
                 .build();
             return new WebAuthnServer(webAuthnCredentialRepository,
-                newCache(WEBAUTHN_SERVER_CACHE_SIZE), newCache(WEBAUTHN_SERVER_CACHE_SIZE), relyingParty,
+                new WebSessionWebAuthnCache<>("WebAuthn_registerRequestStorage", RegistrationRequest.class),
+                new WebSessionWebAuthnCache<>("WebAuthn_assertionRequestStorage", AssertionRequestWrapper.class),
```

Comment:
Would you mind turning these into proper beans? Particularly conditional / refreshable beans, so they can be supplanted if necessary?

Reply:
OK. Done